.gitignore

```diff
@@ -23,4 +23,4 @@ cedar-drt/fuzz/artifacts
 
 # We expect a copy of `cedar-policy/cedar` present in this directory, but do
 # not track it in version control.
-cedar/
+/cedar-policy/cedar
```

Comment:
Probably don't want to change this since CI expects the `cedar/` path

Reply:
Ah, so this line messes up stuff on macos because of case sensitivity; i can restore it but it just means the entire cedar-lean/cedar directory is ignored when i commit. i can just manually change it back and forth for now but it could be worth resolving